R/clean_names.R

```diff
@@ -119,6 +120,20 @@ clean_names.tbl_graph <- function(dat, ...) {
   dplyr::rename_all(dat, .funs=make_clean_names, ...)
```

Comment:
Unrelated, but can you change this to `rename_with` instead of `rename_all` since `rename_with` is the currently-preferred version?

Reply:
I tried changing this line to `dplyr::rename_with(dat, janitor::make_clean_names, .cols = dplyr::everything(), ...)` and I got a failed test:
```
-- Error (test-clean-names.R:596:3): tbl_graph/tidygraph -----------------------
Error: no applicable method for 'rename_with' applied to an object of class "c('tbl_graph', 'igraph')"
```
I like the idea of updating this now-superseded function.  Do you see a reasonable way forward or should we just punt on this one for now?

Reply:
I would leave it alone for now.  I don't see a clean way to fix it.